src/plugins/data-ajax/reports/pre-a11y-1.json

```
@@ -0,0 +1,589 @@
+{
+    "@context": "https://wet-boew.github.io/vocab/context/2023/assessment-report-en.json-ld",
+
+    "@type": ["earl:Assertion", "acr:AssessmentReport"],
+
+    "earl:subject": {
+        "@id": "_:subject",
+        "dct:references": "https://wet-boew.github.io/gcweb-compiled-demos/wetboew-demos/data-ajax/data-ajax-en.html",
+        "@type": [
+            "earl:TestSubject",
+            "schema:WebPage"
+        ],
+        "dct:description": "Accessibility conformance reports of Data Ajax",
+        "earl:pointer": {
+            "@type": "oa:CssSelector",
+            "@value": "data-ajax-after,data-ajax-append,data-ajax-before,data-ajax-prepend,data-ajax-replace,#filter-id,filter-selector"
```

This property must be a valid CSS selector as specified by the type [oa:CssSelector](https://www.w3.org/TR/annotation-model/#css-selector)

Here I think you wanted to select some data attribute and class.
```suggestion
            "@value": "[data-ajax-after],[data-ajax-append],[data-ajax-before],[data-ajax-prepend],[data-ajax-replace],#filter-id,.filter-selector"
```
The modified CSS selector that I suggest do select 15 nodes in the test subject.

---

done